_docs/style/text-conventions/general-guidelines.md

```diff
@@ -0,0 +1,194 @@
+---
```

Comment:
thinking about all of the the examples we present throughout text conventions. should we avoid obviously .com marketing examples? e.g. below " We offer you a comprehensive portfolio of hosting options." that's not really good for UI...

Reply:
great point. I'm combing through a control panel UI for better examples.